src/containers/SortingView.tsx

```diff
@@ -181,7 +181,10 @@ const SortingView: React.FunctionComponent<Props> = (props) => {
     return <h3>{`Recording not found: ${sorting.recordingId}`}</h3>
   }
 
-  const selectedUnitIdsLookup: {[key: string]: boolean} = selection.selectedUnitIds.reduce((m, uid) => {m[uid + ''] = true; return m}, {} as {[key: string]: boolean})
+  const sv = plugins.sortingViews['MVSortingView']
```

Comment:
Now the SortingView displays only the MVSortingView as opposed to showing the accordion view of the plugins. Note that those plugin views are all available within the MVSortingView.